cscs-checks/tools/profiling_and_debugging/ipm_mpi_omp.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.14')
+@rfm.parameterized_test(['C'], ['C++'], ['F90'])
+class Ipm(rfm.RegressionTest):
+    def __init__(self, lang):
+        super().__init__()
+        self.name = 'IPM_%s' % lang.replace('+', 'p')
+        self.descr = '%s check' % lang
+        self.language = lang
```

Comment:
The `self.language` is not used anywhere so delete this line.

Reply:
It is used below: `if lang == 'F90':`

Reply:
ah, i understand now